src/Processor/DateProcessor.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the BibTex Parser.
+ *
+ * (c) Renan de Lima Barbosa <renandelima@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace RenanBr\BibTexParser\Processor;
+
+
+use DateTime;
+
+class DateProcessor
+{
+    use TagCoverageTrait;
```

Comment:
I don't think this trait is required here, because this processor won't change any tag, but rather create a new one.

BTW we could use `TagSearchTrait` to make sure we find the tag we depend on.